editor/inspector_dock.cpp

```diff
@@ -446,8 +461,19 @@ void InspectorDock::update(Object *p_object) {
 	p->clear();
 	p->add_icon_shortcut(get_icon("GuiTreeArrowDown", "EditorIcons"), ED_SHORTCUT("property_editor/expand_all", TTR("Expand All")), EXPAND_ALL);
 	p->add_icon_shortcut(get_icon("GuiTreeArrowRight", "EditorIcons"), ED_SHORTCUT("property_editor/collapse_all", TTR("Collapse All")), COLLAPSE_ALL);
-	p->add_separator();
 
+	p->add_separator(TTR("Property Name Style"));
+	p->add_radio_check_item(TTR("Raw"), PROPERTY_NAME_STYLE_RAW);
```

Comment:
I feel "Raw" might not be very explicit. Maybe "Raw Path" or just "Path"/"Property Path"?

Reply:
It's only part of a path, i.e. there is no slash in it, more like a snake cased identifier. I think it's confusing to call it Path style.